### ISSUE TITLE
`empty-object` complains about lambda object inside atom

### ISSUE THREAD

**Issue by h1alexbel** [objectionary/lints#412]
This is the atom object in XMIR:

```xml
<o name="@">
  <o name="λ"/>
</o>
```

This is what I'm getting:

```text
The object "λ" is empty. It doesn't have any attributes, neither void nor attached (empty-object)
```

`empty-object` should not complain about lambda object inside atom.

**Comment by h1alexbel** [comment]
@yegor256 looks like new bug

**Comment by Marat-Tim** [comment]
Let me try to fix it

**Comment by 0crat** [comment]
@h1alexbel Hey there, bug hunter! 🐛 You've just scored 8 points for spotting that bug - nice work! Your sharp eye is helping us level up our project's quality. Got any more [quirks](https://www.yegor256.com/2018/02/06/where-to-find-more-bugs.html) you've noticed? [Don't hold back](https://www.yegor256.com/2014/04/13/bugs-are-welcome.html), [let us know](https://www.yegor256.com/2018/04/24/right-way-to-report-bugs.html)! By the way, your current score is [+4](https://www.eolang.org/zerocracy/objectionary-vitals.html). Keep those reports coming!

**Comment by VasilevNStas** [comment]
@yegor256 @volodya-lombrozo @maxonfjvipon, this bug is **already fixed on `master`**. The lambda-inside-atom case was fixed back in March 2025 by @Marat-Tim:
- `f1acdc91` — "removed the empty object warning for lambda objects",
- `5b02ba24` — "fix tests",
with a dedicated pack test `packs/single/empty-object/allows-lambda-object-in-atom.yaml` (asserts zero defects).

The `allows-lambda-object-in-atom` test passes green in the current build (`LtByXslTest`, 383 tests, 0 failures). Since the reported false-positive no longer reproduces, I'd suggest closing this issue.

**Comment by volodya-lombrozo** [comment]
@VasilevNStas Indeed, thank you. We even have a test for it: https://github.com/objectionary/lints/blob/master/src/test/resources/org/eolang/lints/packs/single/empty-object/allows-lambda-object-in-atom.yaml
